README.md

```diff
@@ -50,6 +50,8 @@ You will need to use Node.js `Buffer` API compatible, if you are running inside
   digest: <Buffer 0b ee c7 b5 ea 3f 0f db c9 5d 0d d4 7f 3c 5b c2 75 da 8a 33> }
 ```
 
+For encoding and decoding, the [bs58 module](https://github.com/cryptocoinjs/bs58) is recommended.
```

Comment:
Apparently there are two encodings for base-58, one used by Flickr and one used by Bitcoin. There was some confusion when trying to use this module about how exactly to encode hashes so they would be compatible with hashpipe and (potentially) IPFS.

If there are other supported encodings, they should be documented as well.

I've poked around the go-multihash code a bit and it seems much more developed than the JS version.